src/components/Profile/ProfileBanner.tsx

```diff
@@ -37,27 +40,22 @@ const ProfileBanner = ({user}: {user: User | undefined}) => {
 
     return (
         <>
-            <Modal isOpen={editPermissionModal} justifyContent='center' alignItems='center'
-                onClose={() => setEditPermissionModal(false)} height='fill' position='fixed'>
-                <Modal.Content maxWidth='400px'>
-                    <Modal.CloseButton/>
-                    <Modal.Header>Edit User Permissions</Modal.Header>
-                    <Modal.Body>
-                        <Select defaultValue={'' + user?.status}>
-                            <Select.Item label='Unverified' value='0'/>
-                            <Select.Item label='Can log in' value='1'/>
-                            <Select.Item label='Can post' value='2'/>
-                            {userAuth === 4 && <Select.Item label='Employee' value='3'/>}
-                        </Select>
-                        <Button onPress={handleEditPermission} marginTop='1'>
-                            <Text variant='button'>Confirm</Text>
-                        </Button>
-                    </Modal.Body>
-                    
-                </Modal.Content>
-            </Modal>
+            <Popup open={editPermissionModal} closeOnDocumentClick={false} 
+                onClose={() => setEditPermissionModal(false)} modal>
+                <Box p={4} w='100%' h='100%' borderRadius='sm'>
+                    <Text alignSelf='center' bold fontSize='lg'>Edit user permission level</Text>
+                    <Select defaultValue={'' + user?.status}>
+                        <Select.Item label='Unverified (cannot login)' value='0'/>
+                        <Select.Item label='Verified (can login, but cannot post)' value='1'/>
+                        <Select.Item label='Approved (can post)' value='2'/>
+                        {userAuth === 4 && <Select.Item label='Employee' value='3'/>}
```

Comment:
Do we want to add the manager switch in this popup as well?

Reply:
Yeah we should, but I think I might want to reformat the selection process to make it more intuitive. I'm gonna work on it for a bit then ping you when I want you to look over it.